store/README.md

```diff
@@ -40,6 +40,24 @@ provide the public API for creating, deleting, updating, and querying instances
 within this model. They also provide read/write transactions which have 
 _serializable isolation_ within the `Store` scope.
 
+*Indexes*
```

Comment:
Some information about indexes here. We _could_ also include a benchmark run, but since this is so variable, I thought I caveat/informational blurb here instead was more appropriate.